base/applications/network/ipconfig/lang/ro-RO.rc

```diff
@@ -82,10 +82,10 @@ END
 
 STRINGTABLE
 BEGIN
-    IDS_DNSNORECORD         "\tNo records of type %ls\n\n"
-    IDS_DNSNONAME           "\tName does not exist.\n\n"
-    IDS_DNSFLUSHERROR       "Could not flush the DNS Resolver Cache: "
-    IDS_DNSFLUSHSUCCESS     "Successfully flushed the DNS Resolver Cache.\n"
+    IDS_DNSNORECORD         "\tNu sunt înregistrări de tipul %ls\n\n"
+    IDS_DNSNONAME           "\tNumele nu există.\n\n"
+    IDS_DNSFLUSHERROR       "Nu a putut fi golită memoria de depozitare a rezolvatorului DNS: "
+    IDS_DNSFLUSHSUCCESS     "Memoria de depozitare a rezolvatorului DNS a fost golită cu succes.\n"
```

Comment:
I'm not keen on literal translation of certain English terminologies like this one. I would argue `Memoria de depozitare a resolverul DNS` is more fitting imho but I'm eager to hear @reactos-contributor's thoughts.

Reply:
It's not easy to strike a balance between what may sound familiar (i.e. the English term one may have gotten used to) and a term that attempts to respect the destination language. In this specific case however, I don't really have room for a choice on translating "DNS Resolver" anymore, since I have a history of contribution where I've already chosen (although I forgot about it). Here is this very file from one of my PRs touching it (five years ago): https://github.com/reactos/reactos/blob/31019e8c0cbee14b8a6f3e30980bf0d95c175309/base/applications/network/ipconfig/lang/ro-RO.rc#L48